README.md

```diff
@@ -1,3 +1,7 @@
 [![Build Status](https://travis-ci.org/IBM/MAX-Base.svg?branch=master)](https://travis-ci.org/IBM/MAX-Base)
 
-This is a base image for IBM Model Asset Exchange
+This repository contains code for the "base" image used by all other
+[Model Asset Exchange](https://developer.ibm.com/exchanges/models/) models.
+
+The base image is intended to contain requirements common across all MAX models to reduce the amount of packages that
+need to be downloaded when downloading multiple MAX models.
```

Comment:
"when downloading or building multiple containers for MAX models"?

Reply:
After you download 1 max model subsequent models will download faster because you already have the first couple of layers.

Reply:
Sorry I wasn't clear - was suggesting perhaps to change the sentence:

" .. when downloading ..." -> "... when downloading or building ..."

Just since the user may be building models locally, technically they are not downloading the pre-built model containers.

Very minor though